scripts/assemble_changelog.py

```diff
@@ -219,8 +219,10 @@ def add_categories_from_text(self, filename, line_offset,
                                        category.name.decode('utf8'))
 
             body_split = category.body.splitlines()
+            re_has_url = re.compile('.*http[s]?://.*')
```

Comment:
If you bother to precompile the regex, make it a class element like the others.

Reply:
This is not a class method.
I commited new change with refering to the compiled regex via self.__class__
Please comment on the new commit if this is acceptable, or if I should move the compiled regex back to the method definition.

Reply:
refering to class members from non-class method is not allowed by pylint.
Moved compiled regexes to method defintion

Reply:
> refering to class members from non-class method is not allowed by pylint

I don't understand what you're referring to. `add_categories_from_text` is an instance method and can refer to class members. All class members are members of instances as well.

Reply:
This is still pending.